### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by youmari**
- Modify the page title.
- Modify the alt text of the project's image and icons of skills cards
- Text heading no issues found.
- Color contrast  no issues found (using the Figma design)
- Resize, interaction, Moving content, multimedia, : no issues found 👍 
- The basic structure of the page: is well structured

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./css/style.css">
-    <title>Welcome to my Portfolio</title>
+    <title>my Portfolio Homepage</title>
```
Comment (Mihndim2020) [inline index.html:11]: - [ ] To make your site more accessible, please make the page title adequately and briefly describes the content of the page. For example `**Mark's portifolio**,  **Mark full-stack webdeveloper**. Please kindly fix that.
-
youmi

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @youmari, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Good job passing linter checks

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (youmari) [reply index.html]: done thanks

**Comment by youmari** [comment]
@Mihndim2020 you forgot to reset the button in the dashboard to send a request for review

**Pull request merged**